xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/Translations.xml

```diff
@@ -40,30 +40,30 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>annotations.title=Annotate
-annotations.menu.loading=Loading annotations settings
+annotations.menu.loading=Loading annotation settings
 annotations.menu.loaderror=Failed: 
-annotations.tab.info.noannotations=No annotations for this page
+annotations.tab.info.noannotations=No annotations on this page
 
 annotations.settings.display=Show annotations 
-annotations.settings.error.wrongsyntax=Annotations are not available for pages in XWiki/1.0 syntax.
+annotations.settings.error.wrongsyntax=Annotations are not available for pages written in XWiki/1.0 syntax.
 annotations.settings.error.notarget=No page specified to get annotations settings for.
 
 annotations.annotated.loading=Loading annotated page
 annotations.annotated.loaderror=Failed: 
 annotations.annotated.loaderror.wrongresponse=Wrongly formatted server response
 annotations.annotated.error.noannotatedelement=Annotations could not be loaded because the content is not available.
-annotations.annotated.error.wrongsyntax=Annotations are not available for pages in XWiki/1.0 syntax.
+annotations.annotated.error.wrongsyntax=Annotations are not available for pages written in XWiki/1.0 syntax.
 
 annotations.action.edit.text=[Edit]
-annotations.action.edit.tooltip=Edit this annotation
+annotations.action.edit.tooltip=Edit annotation
 annotations.action.edit.submit.text=Update
 annotations.action.edit.cancel.text=Cancel
-annotations.action.edit.success=Annotation has been successfully updated.
+annotations.action.edit.success=Annotation updated!
```

Comment:
I'm not a fan of using exclamation marks. The tone should be informative, calm and assertive. The users should not feel that you are shouting at them or that you are over excited about a regular event. Please use a comma instead. The rest of the change, that simplifies the phrase, is good.

Reply:
Done.